specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/transfers.json

```diff
@@ -0,0 +1,1182 @@
+          },
+          "default": {
+            "description": "Unexpected error.",
+            "schema": {
+              "$ref": "./types.json#/definitions/ArmErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Billing/transfers/{transferName}/decline": {
+      "post": {
+        "tags": [
+          "RecipientTransfers"
+        ],
+        "summary": "Declines a transfer request.",
+        "operationId": "RecipientTransfers_Decline",
+        "x-ms-examples": {
+          "DeclineTransfer": {
+            "$ref": "./examples/DeclineTransfer.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/transferNameParameter"
+          },
+          {
+            "$ref": "./types.json#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Details of the declined transfer.",
+            "schema": {
+              "$ref": "#/definitions/RecipientTransferDetails"
+            }
+          },
+          "default": {
+            "description": "Unexpected error.",
+            "schema": {
+              "$ref": "./types.json#/definitions/ArmErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Billing/transfers/{transferName}": {
+      "get": {
+        "tags": [
+          "RecipientTransfers"
+        ],
+        "summary": "Gets a transfer request by ID. The caller must be the recipient of the transfer request.",
+        "operationId": "RecipientTransfers_Get",
+        "x-ms-examples": {
+          "RecipientTransferGet": {
+            "$ref": "./examples/GetRecipientTransfer.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/transferNameParameter"
+          },
+          {
+            "$ref": "./types.json#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Details of the transfers with given Id.",
+            "schema": {
+              "$ref": "#/definitions/RecipientTransferDetails"
+            }
+          },
+          "default": {
+            "description": "Unexpected error.",
+            "schema": {
+              "$ref": "./types.json#/definitions/ArmErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Billing/transfers": {
+      "get": {
+        "tags": [
+          "RecipientTransfers"
+        ],
+        "summary": "Lists the transfer requests received by the caller.",
+        "operationId": "RecipientTransfers_List",
+        "x-ms-examples": {
+          "RecipientTransfersList": {
+            "$ref": "./examples/ListRecipientTransfers.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "./types.json#/parameters/apiVersionParameter"
+          },
+          {
+            "name": "$filter",
```

Comment:
ARM guidelines ask us to support OData query parameters without the $ (ie. `filter`, `search`, `orderBy` etc...)